pinot-common/src/main/java/org/apache/pinot/common/function/scalar/DataTypeConversionFunctions.java

```diff
@@ -32,6 +33,23 @@ public class DataTypeConversionFunctions {
   private DataTypeConversionFunctions() {
   }
 
+  @ScalarFunction
+  public static Object cast(Object value, String targetType) {
+    if (value == null) {
```

Comment:
(minor) both arguments won't be `null` here since the function is not annotated as `nullableParameters`

Reply:
thanks, fixed